packages/gatsby-theme-carbon/src/components/SquareCard/square-card.scss

```diff
@@ -0,0 +1,185 @@
+// Square Card Group
+.square-card-group .#{$prefix}--col-lg-4 {
+  margin-bottom: 0;
+}
+.square-card-group .#{$prefix}--square-card {
```

Comment:
We want to use CSS modules for new components. We have some [contribution notes](https://gatsby.carbondesignsystem.com/contributions#sass-and-css-modules) on CSS modules, the other components can help for reference too.

Reply:
Oh ok. I was referencing how the other cards were built

Reply:
Ah that makes sense, those were old components we never migrated over. Let me know if you run into any issues or wanna pair on some of it